src/Components/App/App.js

```diff
@@ -34,9 +36,9 @@ class App extends Component {
   }
 
   addUser = newUser => {
-    console.log(newUser)
-    this.setState({ user: newUser })
-
+    console.log(newUser);
+    this.setState({ isLoading: true });
+    setTimeout(() => this.setState({ user: newUser, isLoading: false }), 7000)
```

Comment:
Beautiful.  Love how we're adding some state to render this.  I think we can make it a goal for tomorrow to conditionally render once our fetch is resolved, but if not, we can put it in as a known bug/future iteration.